bin/ExtractGName.R

```diff
@@ -7,7 +10,7 @@ zip.fname <- paste(fname, 'zip', sep='.')
 heatmap.dat <- file.path(fname, 'heatmap.RData')
 load(unz(zip.fname, heatmap.dat))
 
-gene.list <- strsplit(go.list[[1]], ':')
-gname.list <- sapply(gene.list, function(x) x[1])
+gene.list <- as.data.frame(go.list, stringsAsFactors = FALSE)
+gname.list <- str_split_fixed(gene.list[,1],':',2)[,1]
```

Comment:
I have a further modified version which I use for extracting clusters. I tried to keep this patch as close to the original as possible, just updating it to keep functionality.